### ISSUE TITLE
Compatibility with R version 3.6.0

### ISSUE THREAD

**Issue by calebjpicker** [eddelbuettel/rquantlib#148]
I'm getting an error message stating that RQuantLib is not available for R version 3.6.0.

**Comment by eddelbuettel** [comment]
Pretty please, can you make the report meaningful?  _What operating system?_ 

See https://cloud.r-project.org/web/packages/RQuantLib/index.html -- looks like Windows and macOS have binaries.  Source is obviously there.

**Comment by calebjpicker** [comment]
Thank you for your reply.  I've installed RStudio on Amazon Linux 2, running an EC2 instance.  (I'm fairly new to all of this, so I apologize!).

Here is my version of RStudio:

![image](https://user-images.githubusercontent.com/50608203/78145288-8ab45300-73e5-11ea-9454-16bec1ff8c9f.png)

And here is my OS:

![image](https://user-images.githubusercontent.com/50608203/78145366-a4559a80-73e5-11ea-8709-e8f502d71527.png)

Please let me know whethere there's more information I can provide.

**Comment by eddelbuettel** [comment]
Please talk to *RStudio*.  You are a lacking *a premade binary* from their channels. 

As I told you, R has binaries.  In R, one installs via `install.packages("RQuantLib")`.  Not a bug in this package, so closing.

**Comment by eddelbuettel** [comment]
See StackOverflow and other places for answers to questions such as "how do I install a R package from source on platform X" or ditto for "how do I install a binary package".  

Also, and as the README.md at GitHub says, we have a mailing list so for general usage questions of "how do I ..." or "why can't I ..." please try it.  It's a helpful venue.  This issue ticket is more for actual shortcomings and bugs which this one isn't -- CRAN has packages for all OSs for the current version.

**Comment by eddelbuettel** [comment]
One last (personal) comment, as you choose a cloud deployment (good) with Linux as a base (good).  You problem, is essence, is that you cannot find a pre-made binary, and don't (yet?) know how to build from source.

AWS Linux comes with all the weight of AWS, but it still "sucks" relative to alternatives because of (relative) paucity of binaries.  For e.g. Ubuntu 18.04 you would have a choice of a) several hundred CRAN packages directly in the distribution, including r-cran-rquantlib, plus of course the immense PPA adding another 4000+ binary packages.  The geospatial folks just had a nice blog post on this:
https://geocompr.github.io/post/2020/installing-r-spatial-ubuntu/  (which I helped proof-read). 
And of course RStudio runs the same way there. 

So in short, your issue is related to your choice of platform.  Hope this helps a little.

**Comment by calebjpicker** [comment]
Thank you so much.  I am looking into other platforms now.  I will create an Ubuntu 18.04 instance and install from there.  I started with AWS because there was an easy, free one-click template creation, but I didn't realize that it was so lacking.  I appreciate your help.

**Comment by eddelbuettel** [comment]
Glad to have been of help.  One thing, though, is still a bug somewhere.  You are on Linux, which _generally_ defaults to source installations.  It should not have told you 'no RQuantLib for R 3.6.0'.  But that is somebody else's bug.

BTW happy Ubuntu user here too.  I run 19.10 now, and 20.04 probably by June (give the April release a few weeks. I have an 18.04 instance on AWS as well for work-related matters.)  That said, it all depends.  Other people are happy with Fedora too, or even (shudder ;-) ) on other OSs....

**Comment by calebjpicker** [comment]
Quick Question:
Can you direct me to resources on how to install a web interface on AWS
Ubuntu server for RStudio?

**Comment by eddelbuettel** [comment]
I can't, and I don't even understand the question.  Can you maybe think about rephrasing what you want, then search for possible clues and if that fails, ask on StackOverflow.

If you meant an overall control panel for a Linux box, consider https://askubuntu.com/ which I use too at times.  

But as we are now far from RQuantLib issues, let's stop this here.
